misc/dist/macos/editor_info_plist.template

```diff
@@ -5,21 +5,21 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleExecutable</key>
-	<string>Godot</string>
+	<string>Blazium</string>
 	<key>CFBundleName</key>
-	<string>Godot</string>
+	<string>Blazium</string>
 	<key>CFBundleIconFile</key>
-	<string>Godot.icns</string>
+	<string>Blazium.icns</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.godotengine.godot</string>
+	<string>org.blaziumengine.blazium</string>
```

Comment:
Should this be `app.blaziumengine.blazium` if going by domain name derived conventions?

Reply:
See idk... :| I'm waiting to get access to apple, to see if I set this value there, and shit. I relaly don't know @.@

Reply:
isn't this similar to how android works where it goes com.{studio name}.{project name}?
idk about mac, but my guess is that it's just their package convention

Reply:
After messing with some stuff on the Android side, I think it should be changed to use `app.blaziumengine`